lib/init.coffee

```diff
@@ -6,13 +6,13 @@ module.exports =
     executablePath:
       title: 'Executable Path'
       type: 'string'
-      default: 'pep257'
-      description: "Path to executable pep257 cmd."
+      default: 'pydocstyle'
```

Comment:
@julesjulian if you have time/interest in writing a warning, go for it! I don't have much coffeescript/javascript experience, so it'd be a non-trivial amount of time for me to include that in this PR. 

If we're going to go down this route, I'd suggest pushing out a final release as `pep257` that includes the warning before changing the package name.

I'm not sure which URL Atom's package manager is tied to--the atom.io directory or directly to GitHub. If GitHub, then when we change the package name, GitHub will silently redirect those requests to the new URL. OTOH, if it's tied to atom.io directory, no idea whether atom.io implements redirects if packages change names. Perhaps @Arcanemagus knows more about the internals here?